.changeset/ten-dancers-do.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+Update unenv to the unenv-nightly@2.0.0-1725621861.f1c0bb8
```

Comment:
This needs updating to match the version update you just force pushed

Reply:
Good catch, thanks!